lib/services/ios-project-service.ts

```diff
@@ -157,10 +158,14 @@ export class IOSProjectService extends projectServiceBaseLib.PlatformProjectServ
 				basicArgs.push("-target", this.$projectData.projectName);
 			}
 
+			// Starting from tns-ios 1.4 the xcconfig file is referenced in the project template
+			if (semver.lt(this.platformData.frameworkVersion, "1.4.0")) {
```

Comment:
is it possible to get here in case the framework version is undefined? For example:
```
tns create app1
cd app1
tns build ios
```
In this case I'm not sure that the moment when we set frameworkVersion property, it will have any value.
Maybe just for security we can change the if:
```TypeScript
if (this.platformData.frameworkVersion && semver.lt(this.platformData.frameworkVersion, "1.4.0")) 
```